src/core/segment_buffers/implementations/text/html/text_track_cues_store.ts

```diff
@@ -26,6 +26,11 @@ import {
   removeCuesInfosBetween,
 } from "./utils";
 
+const DELTA_CUES_GROUP = 1e-3; // 1ms
+
+// segment_duration / RELATIVE_DELTA_RATIO = relative_delta
+// relative_delta is the tolerance to determine if two segements are the same
+const RELATIVE_DELTA_RATIO = 5;
```

Comment:
We generally set such "JSDoc-y" presentation of variables, constants and functions with a `/**` JSDoc prefix:
```js
/**
 * This is thing, it is useful when ...
 */
const MY_THING = 525;
```

This then seems to be properly picked up by programming tools to explain the corresponding symbol (e.g. I know that on my vim, ts-server only picks up JSDoc comments to document suggested variables. I would guess it's the same story in VSCode?)